<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginationWidget</class>
 <widget class="QWidget" name="PaginationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="spinPage">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMax">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboPerPage"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
